README.md

```diff
@@ -46,16 +55,27 @@ chown www-data:www-data -R *
 npm install
 npm run build
 cd statistics
-chmod +x update.sh
-./update.sh .
+#chmod +x update.sh     #OLD
+#./update.sh .          #OLD
```

Comment:
These commands have been commented out here, but I guess the webhook still runs on updates to this repo. Has those commands been changed in that script as well?

Reply:
As mentioned above I've sent Spidera to look at it!

Reply:
The webhook is no longer active since devil was made the production server in 2020. I will set a new one up for this pull request.